src/djcytoscape/forms.py

```diff
@@ -37,12 +33,7 @@ class Meta:
 
     name = forms.CharField(max_length=50, required=False, help_text="If not provided, the initial quest's name will be used")
     
-    initial_content_object = ContentObjectChoiceField(
-        label='Initial Object',
-        required=True,
-        queryset=QuerySetSequence(*[klass.objects.all() for klass in get_model_options()]),
-        widget=ContentObjectSelect2Widget(search_fields=["name__icontains"]),
-    )
+    initial_content_object = AllowedContentObjectChoiceField(label='Initial Object')
```

Comment:
This will cause errors, especially in a new database, because content_types framework doesn't exist yet, but the field is being initialized when the code is read, and the __init__ above is trying to access ContentType